osu.Game.Tests/Visual/Online/TestSceneVotePill.cs

```diff
@@ -21,6 +22,11 @@ public class TestSceneVotePill : OsuTestScene
             typeof(VotePill)
         };
 
+        protected override bool UseOnlineAPI => true;
```

Comment:
why are we using online api now? this is going to be triggering login attempts on the actual site, no? which could get IPs blocked and trigger security countermeasures.

how about fixing `IsLoggedIn` in `DummyAPIAccess` instead rather than working around the true issue.